packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-testing.md

```diff
@@ -3,83 +3,79 @@ id: Testing your accessibility
+### 5. Test with a screen reader 
 
-- **Landmarks**: Landmarks help communicate the structure of a page by identifying regions. There are many types of landmarks, such as banners, navigation, main, and form. When there are more than one of the same role (for example, two navigation regions on the same page), these regions should be differentiated by an `aria-label`.
+You can use any screen reader that is available in your operating system. To test PatternFly, we target the following screen readers:
+  - JAWS with Chrome, Windows ([JAWS keyboard shortcuts](//dequeuniversity.com/screenreaders/jaws-keyboard-shortcuts))
+  - Voiceover with Safari, Mac ([Voiceover keyboard shortcuts](//dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts))
+  - NVDA with Firefox, Windows ([NVDA keyboard shortcuts](//dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts))
 
-- **Dynamic content**: One of the biggest accessibility concerns with dynamic content is the need to notify users that the content has changed. Sighted users benefit from highlighting or drawing visual attention to the changes. Non-sighted users need to be notified in another way, such as loading a new page, sending the focus to the new content, or using ARIA live announcements.
+### 6. Check color contrast
 
-- **Unrelated notifications**: Notifications from the operating system outside of the web app can interrupt a user while interacting with a page (for example, "You received a new chat message."). Consider this possibility when designing and developing for screen readers, and keep any notification messaging concise to limit the interruption.
+Colors should pass the following contrast checks, to ensure that users across the vision spectrum can understand your product:
+    - Text color against background color ([Understanding WCAG 1.4.3](//www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))
+    - Text color against link color ([Technique G183](//www.w3.org/TR/WCAG20-TECHS/G183.html))
+    - Visible boundaries of buttons and form elements against adjacent background color ([Understanding WCAG 1.4.11](//www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html))
 
+## Accessibility testing checklist
 
-<br />
+We recommend referencing the following checklist as you assess your product's accessibility.
 
-### Checklist for testing
-These are some of the main areas we check for on the PatternFly team. We recommend that you check for these same areas in your applications, too. 
+These are some of the main areas we check for on the PatternFly team to ensure that components meet consistent accessibility standards, but, to check your complete implementation, we recommend checking these same areas in your products. 
```

Comment:
I'm wondering if we can tweak this sentence a little? Looking at the previous version, that probably needed tweaked as well in terms of what was meant. The PF team uses these criteria when checking a UI for a11y, not necessarily components, since some of these don't really apply to the component level. So for example, the entire section on structure is more of a UI consideration (at least in the way we're talking about it). Having heading structure of a page done correctly doesn't really apply to the component level, but when you're creating a page and potentially using multiple components.